Remotion/Web/Development.WebTesting/Configuration/WebTestSettings.cs

```diff
@@ -57,11 +63,35 @@ public static IWebTestSettings Current
       }
     }
 
-    private static IWebTestSettings CreateDefaultWebTestSettings ()
+    public static IWebTestSettings CreateAppConfigBasedWebTestSettings ()
     {
       var settings = (WebTestConfigurationSection)ConfigurationManager.GetSection("remotion.webTesting");
       Assertion.IsNotNull(settings, "Configuration section 'remotion.webTesting' missing.");
       return settings;
     }
+
+#if !NETFRAMEWORK
+    public static IWebTestSettings CreateAppSettingsJsonBasedWebTestSettings ()
```

Comment:
Is this still json-based?

Reply:
What do you mean with "still json-based"? This convenience setup method uses `appsettings.json` as its main config file which is why I chose that name. It does include other configuration sources, like environment variables, but the JSON one is the main source.

Reply:
quick detail question: the code in the method doesn't do anything to actually have a json dependency, does it? the application developer could use any implementation of the config system, even though most use json?

if so, we should not call the public api json-based since that's not the identifying characteristic of the new config api or our implementation in here.

Reply:
We might want to talk about the dependencies that we bring in with the addition of adding the hosting infrastructure. It does have a dependency on System.Text.Json for JSON parsing. We provide a default here, if you want to use any kind of other custom JSON parsing, settings, configuration sources, etc. you will have to call this method yourself.